CSharpMath.Forms.Tests/CSharpMath.Forms.Tests.csproj

```diff
@@ -0,0 +1,6 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <EmbeddedResource Include="..\CSharpMath.Rendering.Tests\ComicNeue_Bold.otf" Link="ComicNeue_Bold.otf" />
```

Comment:
If you're not testing adding custom typefaces, then you should omit this line.

Reply:
After deleting that line at least one of the unit tests failed. Maybe that was not the only thing I changed. I will re-check. (I thought it was related to creating the stream for setting the ImageSource of BaseButton and did not think about it really, I'm sorry.)